### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by YuraFesun**
Create ParcerClass

### CODE REVIEW

File: src/main/java/DataReading/ParserCSV.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ParserCSV {
+
+    public static void parceCSV() {
+        String line = "";
+        String splitBy = ",";
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: hardcode

File: src/main/java/DataReading/ParserCSV.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package DataReading;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ParserCSV {
+
+    public static void parceCSV() {
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: void?

File: src/main/java/DataReading/ParserCSV.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        String line = "";
+        String splitBy = ",";
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(".idea/DataIn.csv"));
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: hardcode

File: src/main/java/DataReading/ParserCSV.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            while ((line = br.readLine()) != null)   //check are we have a next line
+            {
+                String[] ivents = line.split(splitBy);    // use comma as separator
+                System.out.println("Event type" + ivents[0] + ivents[1] + ", Designation=" + ivents[2]
+                        + ", Contact=" + ivents[3]);
+            }
+            br.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: Exception handling?

File: src/main/java/DataReading/ParserCSV.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ParserCSV {
+
+    public static void parceCSV() {
+        String line = "";
+        String splitBy = ",";
+        try {
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: Use try with resources

File: src/main/java/DataReading/ParserCSV.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+            BufferedReader br = new BufferedReader(new FileReader(".idea/DataIn.csv"));
+            while ((line = br.readLine()) != null)   //check are we have a next line
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: Delete comments before push code

File: src/main/java/DataReading/ParserCSV.java [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+            {
+                String[] ivents = line.split(splitBy);    // use comma as separator
```
Comment (AndrewPolishchuc) [inline src/main/java/DataReading/ParserCSV.java]: The same

**Review by AndrewPolishchuc — CHANGES_REQUESTED** [review]
Need to complete your csv file

File: src/main/java/DataReading/ParserCSV.java (hunk 2 above)
Comment (YuraFesun) [reply src/main/java/DataReading/ParserCSV.java]: My bad

File: src/main/java/DataReading/ParserCSV.java (hunk 3 above)
Comment (YuraFesun) [reply src/main/java/DataReading/ParserCSV.java]: gewh

File: src/main/java/DataReading/ParserCSV.java (hunk 4 above)
Comment (YuraFesun) [reply src/main/java/DataReading/ParserCSV.java]: eggg

Comment (YuraFesun) [reply src/main/java/DataReading/ParserCSV.java]: Changed

Comment (YuraFesun) [reply src/main/java/DataReading/ParserCSV.java]: Changed

**Review by AndrewPolishchuc — APPROVED** [review]

**Pull request merged**